src/main/java/com/trolley/trolley/Payment.java

```diff
@@ -26,7 +26,7 @@ public class Payment
     private String fxRate;
     private String memo;
     private String externalId;
-    private Object processedAt;
+    private String processedAt;
```

Comment:
Noticed this one is actually a String in the API / webhooks.
Also wasn't sure if that was intentional or not (if you're planning to change the type)